diff --git a/src/components/ftbot/StrategySelect.vue b/src/components/ftbot/StrategySelect.vue
index 67910559..66691b6e 100644
--- a/src/components/ftbot/StrategySelect.vue
+++ b/src/components/ftbot/StrategySelect.vue
@@ -12,7 +12,7 @@ const emit = defineEmits<{ 'update:modelValue': [value: string] }>();
 
 const botStore = useBotStore();
 
-const strategyCode = computed((): string => botStore.activeBot.strategy?.code ?? '');
+const strategyCode = computed((): string => botStore.activeBot.strategy?.code);
 const locStrategy = computed({
   get() {
     return props.modelValue;
